ui/src/components/Exchange/CreateMarket.css

```diff
@@ -10,6 +10,11 @@
     align-items: flex-end;
 }
 
+.create-market-quantities {
+    margin-top: 10px;
+    width: 250px;
```

Comment:
just make sure when setting widths it wont interfere with responsiveness (as a general reminder but this is a small width so i doubt it will here)

Reply:
👍  Sounds good. For this I just tried it, it looks fine at 500px (and beyond) - which is the smallest width that works before cutting off the token pair boxes. Might be better for the responsiveness if those would go on top of each other when the window is small enough:

![Screen Shot 2020-12-01 at 1 48 26 PM](https://user-images.githubusercontent.com/71082197/100783572-f539ea80-33db-11eb-93ac-62fec45f3926.png)

Reply:
agreed - `flex-wrap: wrap` should do the trick

Reply:
I changed `.create-market-options` to:

```
.create-market-options {
    display: flex;
    flex-direction: row;
    align-items: flex-end;
    flex-wrap: wrap;
}
```

and it works until the icon gets squeezed in and then looks like this:
![Screen Shot 2020-12-01 at 2 02 52 PM](https://user-images.githubusercontent.com/71082197/100785174-4d71ec00-33de-11eb-8001-95c208cb1c2f.png)

If there's a fix for this you can tell me I'll add it to the PR, but my CSS/responsiveness knowledge isn't quite deep enough to know the answer off the top of my head...